lib/workos/user_management.rb

```diff
@@ -513,7 +513,7 @@ def authenticate_user_with_totp(code:, client_id:, pending_authentication_token:
       def enroll_auth_factor(user_id:)
         response = execute_request(
           request: post_request(
-            path: "/users/#{user_id}/auth/factors",
+            path: "/user_management/#{user_id}/auth/factors",
```

Comment:
also `/user_management/users/`

Reply:
(and the below auth factors endpoint too)

Reply:
Good point! I was misreading the API controller, thanks for catching!